patches/expo-image+2.0.4.patch

```diff
@@ -1,3 +1,21 @@
+diff --git a/node_modules/expo-image/android/build.gradle b/node_modules/expo-image/android/build.gradle
+index a85fb5e..ac568c1 100644
+--- a/node_modules/expo-image/android/build.gradle
++++ b/node_modules/expo-image/android/build.gradle
+@@ -44,7 +44,12 @@ dependencies {
+   kapt "com.github.bumptech.glide:compiler:${GLIDE_VERSION}"
+   api 'com.caverock:androidsvg-aar:1.4'
+ 
+-  implementation "com.github.penfeizhou.android.animation:glide-plugin:3.0.2"
++  implementation project(':frameanimation')
++  implementation project(':gif')
++  implementation("com.github.penfeizhou.android.animation:glide-plugin:3.0.2") {
++    exclude group: 'com.github.penfeizhou.android.animation', module: 'frameanimation'
++    exclude group: 'com.github.penfeizhou.android.animation', module: 'gif'
```

Comment:
Without exclusions, it will cause duplicate symbols error.  Since glide-plugin will have frameanimation (required module) in other modules (apng, webp, etc).